lib/pbench/server/api/resources/endpoint_configure.py

```diff
@@ -195,7 +191,9 @@ def get(self):
             }
             response = jsonify(endpoints)
         except Exception:
-            self.logger.exception("Something went wrong constructing the endpoint info")
+            current_app.logger.exception(
```

Comment:
Ideally, this would be an `APIInternalError` now, but this is one of the original APIs that doesn't descend from `ApiBase`. Still, we'd like all of these to get consistent UUIDs and return a message referencing the UUID. I handled a couple of odd cases by constructing an `APIInternalError` and referencing the message property directly for the `abort`.

You don't have to do that here, but clearly I missed some scattered `abort` cases that ought to be fixed. 😦